src/pages/Authentication/Login/Login.jsx

```diff
@@ -127,7 +141,26 @@ export const Login = () => {
 							Forgot your Password?
 						</button>
 					</section>
-					<button className="form__login_btn btn btn--primary">Login</button>
+					<button
+						onClick={onSubmitHandler}
+						className="form__login_btn btn btn--primary"
+					>
+						Login
+					</button>
+					<button
+						onClick={googleSigninHandler}
+						className="form__login_btn form__google btn btn--outline--primary"
+					>
+						{bxIcons.google}
+						<span>Login with Google</span>
+					</button>
+					<button
+						onClick={facebookSigninHandler}
+						className="form__login_btn form__google btn btn--outline--primary"
+					>
+						{bxIcons.facebook}
+						<span>Login with Facebook</span>
+					</button>
```

Comment:
You can extract it in array of objects than you can map over it.

Reply:
Ok. would look into it.